src/components/Grid/Entry/Entry.tsx

```diff
@@ -3,12 +3,23 @@ import PropTypes from 'prop-types';
 
 import styles from './Entry.module.css';
 
+interface EntryProps extends React.HTMLAttributes<HTMLDivElement> {
+  entryPercentWidth?: number;
```

Comment:
Children is required in the original propTypes, but not in the TS interface

Reply:
Also, did we decide we needed to leave the original propTypes and defaultProps, or can they be removed safely now?

Reply:
I believe we agreed to drop them since we are the primary consumers of the library. Plus, trying to have TypeScript and PropTypes together is tricky in certain situations.

Reply:
> Children is required in the original propTypes, but not in the TS interface

I missed this. I can add it back using `Omit`

Reply:
Turns out you don't need to use `Omit` if you are just making a prop required